xbmc/platform/android/activity/JNIXBMCFile.cpp

```diff
@@ -0,0 +1,134 @@
+/*
+ *      Copyright (C) 2018 Christian Browet
+ *      http://xbmc.org
```

Comment:
xbmc.org? ;)

Reply:
Isn't the **XBMC** foundation still "owning" our assets?
All our headers have xbmc.org, afaict

Reply:
No we changed the address to kodi.tv in as many files as we could. Seems some just keep c/p the old headers.